docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -996,33 +996,753 @@ Result:
 └─────────────────────────────────────────────┘
 ```
 
-## reinterpretAsUInt(8\|16\|32\|64)
+## reinterpretAsUInt8
 
-## reinterpretAsInt(8\|16\|32\|64)
+Performs byte reinterpretation similar to [reinterpret_cast](https://en.cppreference.com/w/cpp/language/reinterpret_cast) to type UInt8. 
```

Comment:
C++ examples will only help the minority of users who are familiar with C++.

We should say something like this:

```md
Treats the input value as a value of type UInt8. Unlike `CAST`, the function does not attempt to preserve the original value - if the target type is not able to represent the input type, the output is meaningless.
```